lib/cli/emulate.js

```diff
@@ -20,11 +20,35 @@
  */
 var self,
     open = require('open'),
-    emulate = require('./../server/emulate');
+    emulate = require('./../server/emulate'),
+    help = require('./help'),
+    colors = require('colors');
+
+colors.mode = "console";
 
 self = {
     call: function (opts) {
-        self.start(opts);
+        if (self.validateOptions(opts)) {
+            self.start(opts);
+        } else {
+            console.error(('Options not understood: ' + process.argv.splice(2).join(' ')).red.bold);
+            help.call('emulate');
+        }
+    },
+    validateOptions: function (opts) {
```

Comment:
I was thinking that the command-line processing would actually use it before calling start, but that would require implementing the same functionality in other modules which, in turn, would contaminate this PR with irrelevant changes. Do you want me to make it private? Or, I could finish global command-line option validation and handling.